packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,73 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
 ---
 
 ## Examples
-### Static alert group
-These alerts appear on page load and are discoverable from within the normal page content flow, and will not be announced individually/explicitly to assistive technology.
+
+### Variants 
+
+Alert groups can be the following variants: 
+| Variant | Description |
```

Comment:
This table is not rendering correctly. Maybe it just needs one more empty line before the heading of the table?